clip.py

```diff
@@ -153,14 +156,82 @@ def vectorize(self, payload: ClipInput) -> ClipResult:
 		)
 
 
+class ClipInferenceOpenCLIP:
+	clip_model: open_clip.CLIP | open_clip.CoCa | open_clip.CustomTextCLIP
+
+	def __init__(self, cuda, cuda_core):
+		self.device = 'cpu'
+		if cuda:
+			self.device=cuda_core
+
+		config = {}
```

Comment:
You do not need to declare this variable, it will be created and populated in the `with` block below.

Reply:
oh, didn't know that, thanks!